lua/starfall/libs_sh/entities.lua

```diff
@@ -237,14 +237,30 @@ if CLIENT then
 		local ent = getent(self)
 		if not ent.IsSFHologram and not ent.IsSFProp then SF.Throw("The entity isn't a hologram or custom-prop", 2) end
 
-
 		checkpermission(instance, ent, "entities.setRenderProperty")
 
 		mins, maxs = vunwrap(mins), vunwrap(maxs)
 		ent:SetRenderBounds(mins, maxs)
 		ent.sf_userrenderbounds = {mins, maxs}
 	end
 
+	--- Returns render bounds of the entity as local vectors
+	-- If the render bounds are not inside players view, the entity will not be drawn!
+	-- @client
+	-- @return Vector The minimum vector of the bounds
+	-- @return Vector The maximum vector of the bounds
+	function ents_methods:getRenderBounds()
+		local ent = getent(self)
+
+		local sf_userrenderbounds = ent.sf_userrenderbounds
+		if sf_userrenderbounds then
+			return vwrap(sf_userrenderbounds[1]), vwrap(sf_userrenderbounds[2])
+		end
```

Comment:
GetRenderBounds should return the same thing as this right? Probably don't need this.

Reply:
removed it